tests/roundtrip.rs

```diff
@@ -5,6 +5,8 @@
 #![allow(mixed_script_confusables)]
 
 use byteorder::{LittleEndian, ReadBytesExt, WriteBytesExt};
+#[cfg(feature = "half")]
```

Comment:
- [ ] I think I generally avoid putting `cfg` in tests, especially integration tests because I worry that I could forget to ever actually test it with the feature enabled.
  - You could add `half` to the `required-features` in Cargo.toml for the test.  (doing that could be problematic if `half` was not available on all platforms, but judging from its readme I think it is.)
  - Alternatively, `half` could be tested in a separate test that requires the feature.